README.md

```diff
@@ -27,6 +27,43 @@ function mystery(n) {
 }
 ```
 
+
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+The recurrence relation is $3T(n/3) + n^5$ for $n > 1$ where for $n ≤ 1$, $T(1) = 1$.
+
+This is because when $n ≤ 1$ the funtcion performs a check and returns. For any other value of $n$, the function makes 3 recursive called with argument $n/3$ and goes through 
+a for loop that adds $(n* n) * (n) * (n*n) = n^5$ to the funciton.
+
+$T(n) = 3T(n/3) + n^5$
+
+$= 3(3T(n/3/3) + n^5) + n^5$
+
+$= 3^2 T(n/3^2) + (3+1)n^5$
+
+$= 3(3^2T(n/(3^2)/3) + (3+1)n^5)) + n^5$
+
+$= 3^3 T(n/3^3) + (9+3+1)n^5$
+    
+$= 3^i T(n/3^i) + n^5 \sum_{j=1}^{i} 3^(j-1)$
```

Comment:
Don't understand how you get from here to the next line with the sum. Where does the $n$ in the next line come from?